### PR TITLE
MC test batch - yet another pack refreshed (applications-protocol-http)

### PR DESCRIPTION
**Pull request by omercier**
## Description

yet another pack refreshed

## Target version (i.e. version that this PR changes)

- [ ] 21.10.x (staging)
- [ ] 22.04.x (staging)
- [ ] 22.10.x (staging)
- [ ] 23.04.x (staging)
- [ ] Cloud (staging)
- [x] Monitoring Connectors (staging)
- [ ] 23.10.x (next)

### CODE REVIEW

File: pp/integrations/plugin-packs/procedures/applications-protocol-http.md [hunk 1]
```diff
@@ -71,96 +94,448 @@ It brings the following service templates:
...
+| CRITICALNUMERIC | Critical threshold (Default: on total matching elements)                                                                                                                                                                                                                                                                                   |                                      |             |
+| WARNINGSTRING   | Warning threshold if the string match                                                                                                                                                                                                                                                                                                      |                                      |             |
```
[the remark this answers is not in the document]
Comment (cg-tw) [reply pp/integrations/plugin-packs/procedures/applications-protocol-http.md:231]: matches what?

File: pp/integrations/plugin-packs/procedures/applications-protocol-http.md [hunk 2]
```diff
@@ -71,96 +94,448 @@ It brings the following service templates:
...
+|:----------------|:-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:-------------------------------------|:-----------:|
+| URLPATH         | Set path to get Webpage (Default: '/')                                                                                                                                                                                                                                                                                                     | /                                    |             |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-protocol-http.md:250]:
```suggestion
| URLPATH         | Set path to get web page (Default: '/')                                                                                                                                                                                                                                                                                                     | /                                    |             |
```

File: pp/integrations/plugin-packs/procedures/applications-protocol-http.md [hunk 3]
```diff
@@ -71,96 +94,448 @@ It brings the following service templates:
...
+| CRITICALNUMERIC | Critical threshold (Default: on total matching elements)                                                                                                                                                                                                                                                                                   |                                      |             |
+| WARNINGSTRING   | Warning threshold if the string match                                                                                                                                                                                                                                                                                                      |                                      |             |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-protocol-http.md:261]:
```suggestion
| WARNINGSTRING   | Warning threshold if the string matches                                                                                                                                                                                                                                                                                                      |                                      |             |
```

Comment (cg-tw) [reply pp/integrations/plugin-packs/procedures/applications-protocol-http.md:261]: matches what?

File: pp/integrations/plugin-packs/procedures/applications-protocol-http.md [hunk 4]
```diff
@@ -71,96 +94,448 @@ It brings the following service templates:
...
+| CRITICALSTRING  | Critical threshold if the string match                                                                                                                                                                                                                                                                                                     |                                      |             |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-protocol-http.md:262]:
```suggestion
| CRITICALSTRING  | Critical threshold if the string matches                                                                                                                                                                                                                                                                                                     |                                      |             |
```

Comment (cg-tw) [reply pp/integrations/plugin-packs/procedures/applications-protocol-http.md:262]: matches what?

File: pp/integrations/plugin-packs/procedures/applications-protocol-http.md [hunk 5]
```diff
@@ -71,96 +94,448 @@ It brings the following service templates:
...
+| EXTRAOPTIONS    | Any extra option you may want to add to the command (E.g. a --verbose flag). All options are listed [here](#available-options)                                                                                                                                                                                                                                        |                                      |             |
+
+</TabItem>
+</Tabs>
+
+3. [Deploy the configuration](/docs/monitoring/monitoring-servers/deploying-a-configuration). The service appears in the list of services, and on page **Resources Status**. The command that is sent by the connector is displayed in the details panel of the service: it shows the values of the macros.
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-protocol-http.md:268]:
```suggestion
3. [Deploy the configuration](/docs/monitoring/monitoring-servers/deploying-a-configuration). The service appears in the list of services, and on the **Resources Status** page. The command that is sent by the connector is displayed in the details panel of the service: it shows the values of the macros.
```

File: pp/integrations/plugin-packs/procedures/applications-protocol-http.md [hunk 6]
```diff
@@ -71,96 +94,448 @@ It brings the following service templates:
...
 /usr/lib/centreon/plugins/centreon_protocol_http.pl \
-    --plugin=apps::protocols::http::plugin \
-    --mode=response \
-    --hostname=google.com \
-    --http-backend=curl \
-    --extra-stats \
-    --use-new-perfdata
+	--plugin=apps::protocols::http::plugin \
+	--mode=response \
+	--hostname=10.0.0.1 \
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-protocol-http.md:280]: why 10.0.0.1?

File: pp/integrations/plugin-packs/procedures/applications-protocol-http.md [hunk 7]
```diff
@@ -71,96 +94,448 @@ It brings the following service templates:
...
+|:---------------------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| --hostname           | IP Addr/FQDN of the Webserver host                                                                                                                                                                                                                                                          |
+| --port               | Port used by Webserver                                                                                                                                                                                                                                                                      |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-protocol-http.md:380]:
```suggestion
| --port               | Port used by the web server                                                                                                                                                                                                                                                                      |
```

File: pp/integrations/plugin-packs/procedures/applications-protocol-http.md [hunk 8]
```diff
@@ -71,96 +94,448 @@ It brings the following service templates:
...
+| --proto              | Specify https if needed (Default: 'http')                                                                                                                                                                                                                                                   |
+| --urlpath            | Set path to get Webpage (Default: '/')                                                                                                                                                                                                                                                      |
+| --credentials        | Specify this option if you access webpage with authentication                                                                                                                                                                                                                               |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-protocol-http.md:384]:
```suggestion
| --credentials        | Specify this option if you are accessing  a web page with authentication                                                                                                                                                                                                                               |
```

File: pp/integrations/plugin-packs/procedures/applications-protocol-http.md [hunk 9]
```diff
@@ -71,96 +94,448 @@ It brings the following service templates:
...
+| --password           | Specify password for authentication (Mandatory if --credentials is specified)                                                                                                                                                                                                               |
+| --basic              | Specify this option if you access webpage over basicauthentication and don't want a '401 UNAUTHORIZED' error to be logged on your webserver.  Specify this option if you access webpage over hidden basic authentication or you'll get a '404 NOT FOUND' error.  (Use with --credentials)   |
+| --ntlmv2             | Specify this option if you access webpage over ntlmv2 authentication (Use with --credentials and --port options)                                                                                                                                                                            |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-protocol-http.md:388]:
```suggestion
| --ntlmv2             | Specify this option if you are accessing a web page using ntlmv2 authentication (Use with --credentials and --port options)                                                                                                                                                                            |
```

File: pp/integrations/plugin-packs/procedures/applications-protocol-http.md [hunk 10]
```diff
@@ -71,96 +94,448 @@ It brings the following service templates:
...
+| --timeout            | Threshold for HTTP timeout (Default: 5)                                                                                                                                                                                                                                                     |
+| --no-follow          | Do not follow http redirect                                                                                                                                                                                                                                                                 |
+| --cert-file          | Specify certificate to send to the webserver                                                                                                                                                                                                                                                |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-protocol-http.md:391]:
```suggestion
| --cert-file          | Specify certificate to send to the web server                                                                                                                                                                                                                                                |
```

File: pp/integrations/plugin-packs/procedures/applications-protocol-http.md [hunk 11]
```diff
@@ -71,96 +94,448 @@ It brings the following service templates:
...
+| --key-file           | Specify key to send to the webserver                                                                                                                                                                                                                                                        |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-protocol-http.md:392]:
```suggestion
| --key-file           | Specify key to send to the web server                                                                                                                                                                                                                                                        |
```

File: pp/integrations/plugin-packs/procedures/applications-protocol-http.md [hunk 12]
```diff
@@ -71,96 +94,448 @@ It brings the following service templates:
...
+| --cacert-file        | Specify root certificate to send to the webserver                                                                                                                                                                                                                                           |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-protocol-http.md:393]:
```suggestion
| --cacert-file        | Specify root certificate to send to the web server                                                                                                                                                                                                                                           |
```

File: pp/integrations/plugin-packs/procedures/applications-protocol-http.md [hunk 13]
```diff
@@ -71,96 +94,448 @@ It brings the following service templates:
...
+| --post-param         | Set POST params (Multiple option. Example: --post-param='key=value')                                                                                                                                                                                                                        |
+| --cookies-file       | Save cookies in a file (Example: '/tmp/lwp\_cookies.dat')                                                                                                                                                                                                                                   |
+| --extracted-pattern  | Set pattern to extracted a number (used --warning-extracted and --critical-extracted option).                                                                                                                                                                                               |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-protocol-http.md:401]:
```suggestion
| --extracted-pattern  | Set pattern to extract a number (use --warning-extracted and --critical-extracted options).                                                                                                                                                                                               |
```

File: pp/integrations/plugin-packs/procedures/applications-protocol-http.md [hunk 14]
```diff
@@ -71,96 +94,448 @@ It brings the following service templates:
...
+| --warning-status     | Warning threshold for http response code                                                                                                                                                                                                                                                    |
+| --critical-status    | Critical threshold for http response code                                                                                                                                                                                                                                                   |
+| --warning-time       | Warning threshold in seconds (Webpage response time)                                                                                                                                                                                                                                        |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-protocol-http.md:406]:
```suggestion
| --warning-time       | Warning threshold in seconds (web page response time)                                                                                                                                                                                                                                        |
```

File: pp/integrations/plugin-packs/procedures/applications-protocol-http.md [hunk 15]
```diff
@@ -71,96 +94,448 @@ It brings the following service templates:
...
+|:--------------------------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| --data                    | Set file with JSON request                                                                                                                                                                                                                                                                                                                   |
+| --lookup                  | What to lookup in JSON response (JSON XPath string) (can be multiple) See: http://goessner.net/articles/JsonPath/                                                                                                                                                                                                                            |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-protocol-http.md:422]:
```suggestion
| --lookup                  | What to look up in JSON response (JSON XPath string) (can be multiple) See: http://goessner.net/articles/JsonPath/                                                                                                                                                                                                                            |
```

File: pp/integrations/plugin-packs/procedures/applications-protocol-http.md [hunk 16]
```diff
@@ -71,96 +94,448 @@ It brings the following service templates:
...
+| --lookup-perfdatas-nagios | Take perfdatas from the JSON response (JSON XPath string) Chain must be formated in Nagios format. Ex : "rta=10.752ms;50.000;100.000;0; pl=0%;20;40;; rtmax=10.802ms;;;;"      FORMAT OPTIONS:                                                                                                                                               |
+| --format-lookup           | Take the output message from the JSON response (JSON XPath string) Override all the format options but substitute are still applied.                                                                                                                                                                                                         |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-protocol-http.md:424]: substitute?

File: pp/integrations/plugin-packs/procedures/applications-protocol-http.md [hunk 17]
```diff
@@ -71,96 +94,448 @@ It brings the following service templates:
...
+| --critical-numeric        | Critical threshold (Default: on total matching elements)                                                                                                                                                                                                                                                                                     |
+| --threshold-value         | Which value to use (Default: 'count') Can be: 'values' (only check numeric values)                                                                                                                                                                                                                                                           |
+| --warning-string          | Warning threshold if the string match                                                                                                                                                                                                                                                                                                        |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-protocol-http.md:433]:
```suggestion
| --warning-string          | Warning threshold if the string matches                                                                                                                                                                                                                                                                                                        |
```

Comment (cg-tw) [reply pp/integrations/plugin-packs/procedures/applications-protocol-http.md:433]: matches what?

File: pp/integrations/plugin-packs/procedures/applications-protocol-http.md [hunk 18]
```diff
@@ -71,96 +94,448 @@ It brings the following service templates:
...
+| --critical-string         | Critical threshold if the string match                                                                                                                                                                                                                                                                                                       |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-protocol-http.md:434]:
```suggestion
| --critical-string         | Critical threshold if the string matches                                                                                                                                                                                                                                                                                                       |
```

File: pp/integrations/plugin-packs/procedures/applications-protocol-http.md [hunk 19]
```diff
@@ -71,96 +94,448 @@ It brings the following service templates:
...
+| --unknown-string          | Threshold unknown if the string match                                                                                                                                                                                                                                                                                                        |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-protocol-http.md:435]:
```suggestion
| --unknown-string          | Unknown threshold if the string matches                                                                                                                                                                                                                                                                                                       |
```

File: pp/integrations/plugin-packs/procedures/applications-protocol-http.md (hunk 18 above)
Comment (cg-tw) [reply pp/integrations/plugin-packs/procedures/applications-protocol-http.md:434]: matches what?

File: pp/integrations/plugin-packs/procedures/applications-protocol-http.md (hunk 19 above)
Comment (cg-tw) [reply pp/integrations/plugin-packs/procedures/applications-protocol-http.md:435]: matches what?

File: pp/integrations/plugin-packs/procedures/applications-protocol-http.md [hunk 20]
```diff
@@ -71,96 +94,448 @@ It brings the following service templates:
...
+| --critical-time           | Critical threshold in seconds of webservice response time      HTTP OPTIONS:                                                                                                                                                                                                                                                                 |
+| --hostname                | IP Addr/FQDN of the Webserver host                                                                                                                                                                                                                                                                                                           |
+| --port                    | Port used by Webserver                                                                                                                                                                                                                                                                                                                       |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-protocol-http.md:439]:
```suggestion
| --port                    | Port used by web server                                                                                                                                                                                                                                                                                                                       |
```

File: pp/integrations/plugin-packs/procedures/applications-protocol-http.md [hunk 21]
```diff
@@ -71,96 +94,448 @@ It brings the following service templates:
...
+| --username                | Specify username for authentication (Mandatory if --credentials is specified)                                                                                                                                                                                                                                                                |
+| --password                | Specify password for authentication (Mandatory if --credentials is specified)                                                                                                                                                                                                                                                                |
+| --basic                   | Specify this option if you access webpage over basicauthentication and don't want a '401 UNAUTHORIZED' error to be logged on your webserver.  Specify this option if you access webpage over hidden basic authentication or you'll get a '404 NOT FOUND' error.  (Use with --credentials)                                                    |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-protocol-http.md:445]:
```suggestion
| --basic                   | Specify this option if you are accessing a web page using basic authentication and don't want a '401 UNAUTHORIZED' error to be logged on your web server.  Specify this option if you are accessing a web page using hidden basic authentication or you'll get a '404 NOT FOUND' error.  (Use with --credentials)                                                    |
```

File: pp/integrations/plugin-packs/procedures/applications-protocol-http.md [hunk 22]
```diff
@@ -71,96 +94,448 @@ It brings the following service templates:
...
+| --ntlmv2                  | Specify this option if you access webpage over ntlmv2 authentication (Use with --credentials and --port options)                                                                                                                                                                                                                             |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-protocol-http.md:446]:
```suggestion
| --ntlmv2                  | Specify this option if you are accessing a web page using ntlmv2 authentication (Use with --credentials and --port options)                                                                                                                                                                                                                             |
```

File: pp/integrations/plugin-packs/procedures/applications-protocol-http.md [hunk 23]
```diff
@@ -71,96 +94,448 @@ It brings the following service templates:
...
+| --timeout                 | Threshold for HTTP timeout (Default: 10)                                                                                                                                                                                                                                                                                                     |
+| --cert-file               | Specify certificate to send to the webserver                                                                                                                                                                                                                                                                                                 |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-protocol-http.md:448]:
```suggestion
| --cert-file               | Specify certificate to send to the web server                                                                                                                                                                                                                                                                                                 |
```

File: pp/integrations/plugin-packs/procedures/applications-protocol-http.md [hunk 24]
```diff
@@ -71,96 +94,448 @@ It brings the following service templates:
...
+| --method                 | Specify http method used (Default: 'GET')                                                                                                                                                                                                                                                   |
+| --proto                  | Specify https if needed (Default: 'http')                                                                                                                                                                                                                                                   |
+| --urlpath                | Set path to get webpage (Default: '/')                                                                                                                                                                                                                                                      |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-protocol-http.md:468]:
```suggestion
| --urlpath                | Set path to get web page (Default: '/')                                                                                                                                                                                                                                                      |
```

File: pp/integrations/plugin-packs/procedures/applications-protocol-http.md [hunk 25]
```diff
@@ -71,96 +94,448 @@ It brings the following service templates:
...
+| --credentials            | Specify this option if you access webpage with authentication                                                                                                                                                                                                                               |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-protocol-http.md:469]:
```suggestion
| --credentials            | Specify this option if you are accessing a web page with authentication                                                                                                                                                                                                                               |
```

File: pp/integrations/plugin-packs/procedures/applications-protocol-http.md [hunk 26]
```diff
@@ -71,96 +94,448 @@ It brings the following service templates:
...
+| --username               | Specify username for authentication (Mandatory if --credentials is specified)                                                                                                                                                                                                               |
+| --password               | Specify password for authentication (Mandatory if --credentials is specified)                                                                                                                                                                                                               |
+| --basic                  | Specify this option if you access webpage over basicauthentication and don't want a '401 UNAUTHORIZED' error to be logged on your webserver.  Specify this option if you access webpage over hidden basic authentication or you'll get a '404 NOT FOUND' error.  (Use with --credentials)   |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-protocol-http.md:472]:
```suggestion
| --basic                  | Specify this option if you are accessing a web page using basic authentication and don't want a '401 UNAUTHORIZED' error to be logged on your web server.  Specify this option if you are accessing a web page using hidden basic authentication or you'll get a '404 NOT FOUND' error.  (Use with --credentials)   |
```

File: pp/integrations/plugin-packs/procedures/applications-protocol-http.md [hunk 27]
```diff
@@ -71,96 +94,448 @@ It brings the following service templates:
...
+| --timeout                | Threshold for HTTP timeout (Default: 5)                                                                                                                                                                                                                                                     |
+| --no-follow              | Do not follow http redirect                                                                                                                                                                                                                                                                 |
+| --cert-file              | Specify certificate to send to the webserver                                                                                                                                                                                                                                                |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-protocol-http.md:476]:
```suggestion
| --cert-file              | Specify certificate to send to the web server                                                                                                                                                                                                                                                |
```

File: pp/integrations/plugin-packs/procedures/applications-protocol-http.md [hunk 28]
```diff
@@ -71,96 +94,448 @@ It brings the following service templates:
...
+| --key-file               | Specify key to send to the webserver                                                                                                                                                                                                                                                        |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-protocol-http.md:477]:
```suggestion
| --key-file               | Specify key to send to the web server                                                                                                                                                                                                                                                        |
```

File: pp/integrations/plugin-packs/procedures/applications-protocol-http.md [hunk 29]
```diff
@@ -71,96 +94,448 @@ It brings the following service templates:
...
+| --cacert-file            | Specify root certificate to send to the webserver                                                                                                                                                                                                                                           |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-protocol-http.md:478]:
```suggestion
| --cacert-file            | Specify root certificate to send to the web server                                                                                                                                                                                                                                           |
```

File: pp/integrations/plugin-packs/procedures/applications-protocol-http.md [hunk 30]
```diff
@@ -71,96 +94,448 @@ It brings the following service templates:
...
+| --service-soap     | Service Soap Action (Required)                                                                                                                                                                                                                                                                                                               |
+| --data             | Set file with SOAP request (Required)                                                                                                                                                                                                                                                                                                        |
+| --lookup           | What to lookup in XML response (XPath string) (can be multiple)      FORMAT OPTIONS:                                                                                                                                                                                                                                                         |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-protocol-http.md:498]:
```suggestion
| --lookup           | What to look up in XML response (XPath string) (can be multiple)      FORMAT OPTIONS:                                                                                                                                                                                                                                                         |
```

File: pp/integrations/plugin-packs/procedures/applications-protocol-http.md [hunk 31]
```diff
@@ -71,96 +94,448 @@ It brings the following service templates:
...
+| --critical-numeric | Critical threshold (Default: on total matching elements)                                                                                                                                                                                                                                                                                     |
+| --threshold-value  | Which value to use (Default: 'count') Can be: 'values' (only check numeric values)                                                                                                                                                                                                                                                           |
+| --warning-string   | Warning threshold if the string match                                                                                                                                                                                                                                                                                                        |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-protocol-http.md:506]:
```suggestion
| --warning-string   | Warning threshold if the string matches                                                                                                                                                                                                                                                                                                        |
```

Comment (cg-tw) [reply pp/integrations/plugin-packs/procedures/applications-protocol-http.md:506]: matches what?

File: pp/integrations/plugin-packs/procedures/applications-protocol-http.md [hunk 32]
```diff
@@ -71,96 +94,448 @@ It brings the following service templates:
...
+| --critical-string  | Critical threshold if the string match                                                                                                                                                                                                                                                                                                       |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-protocol-http.md:507]:
```suggestion
| --critical-string  | Critical threshold if the string matches                                                                                                                                                                                                                                                                                                       |
```

Comment (cg-tw) [reply pp/integrations/plugin-packs/procedures/applications-protocol-http.md:507]: matches what?

File: pp/integrations/plugin-packs/procedures/applications-protocol-http.md [hunk 33]
```diff
@@ -71,96 +94,448 @@ It brings the following service templates:
...
+| --warning-time     | Warning threshold in seconds of webservice response time                                                                                                                                                                                                                                                                                     |
+| --critical-time    | Critical threshold in seconds of webservice response time      HTTP OPTIONS:                                                                                                                                                                                                                                                                 |
+| --hostname         | IP Addr/FQDN of the Webserver host                                                                                                                                                                                                                                                                                                           |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-protocol-http.md:510]:
```suggestion
| --hostname         | IP address/FQDN of the web server's host                                                                                                                                                                                                                                                                                                           |
```

File: pp/integrations/plugin-packs/procedures/applications-protocol-http.md [hunk 34]
```diff
@@ -71,96 +94,448 @@ It brings the following service templates:
...
+| --port             | Port used by Webserver                                                                                                                                                                                                                                                                                                                       |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-protocol-http.md:511]:
```suggestion
| --port             | Port used by web server                                                                                                                                                                                                                                                                                                                       |
```

File: pp/integrations/plugin-packs/procedures/applications-protocol-http.md [hunk 35]
```diff
@@ -71,96 +94,448 @@ It brings the following service templates:
...
+| --ntlmv2           | Specify this option if you access webpage over ntlmv2 authentication (Use with --credentials and --port options)                                                                                                                                                                                                                             |
+| --timeout          | Threshold for HTTP timeout (Default: 10)                                                                                                                                                                                                                                                                                                     |
+| --cert-file        | Specify certificate to send to the webserver                                                                                                                                                                                                                                                                                                 |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-protocol-http.md:520]:
```suggestion
| --cert-file        | Specify certificate to send to the web server                                                                                                                                                                                                                                                                                                 |
```

File: pp/integrations/plugin-packs/procedures/applications-protocol-http.md [hunk 36]
```diff
@@ -71,96 +94,448 @@ It brings the following service templates:
...
+| --key-file         | Specify key to send to the webserver                                                                                                                                                                                                                                                                                                         |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-protocol-http.md:521]:
```suggestion
| --key-file         | Specify key to send to the web server                                                                                                                                                                                                                                                                                                         |
```

File: pp/integrations/plugin-packs/procedures/applications-protocol-http.md [hunk 37]
```diff
@@ -71,96 +94,448 @@ It brings the following service templates:
...
+| --cacert-file      | Specify root certificate to send to the webserver                                                                                                                                                                                                                                                                                            |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-protocol-http.md:522]:
```suggestion
| --cacert-file      | Specify root certificate to send to the web server                                                                                                                                                                                                                                                                                            |
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: pp/integrations/plugin-packs/procedures/applications-protocol-http.md [hunk 38]
```diff
@@ -71,96 +94,448 @@ It brings the following service templates:
...
+| --lookup                  | What to lookup in JSON response (JSON XPath string) (can be multiple) See: http://goessner.net/articles/JsonPath/                                                                                                                                                                                                                            |
+| --lookup-perfdatas-nagios | Take perfdatas from the JSON response (JSON XPath string) Chain must be formated in Nagios format. Ex : "rta=10.752ms;50.000;100.000;0; pl=0%;20;40;; rtmax=10.802ms;;;;"      FORMAT OPTIONS:                                                                                                                                               |
+| --format-lookup           | Take the output message from the JSON response (JSON XPath string) Override all the format options but substitute are still applied.                                                                                                                                                                                                         |
```
Comment (omercier) [inline pp/integrations/plugin-packs/procedures/applications-protocol-http.md:424]:
```suggestion
| --format-lookup           | Take the output message from the JSON response (JSON XPath string) Override all the format options but substitute are still applied.                                                                                                                                                                                                         |
```
```suggestion
| --format-lookup           | Take the output message from the JSON response (JSON XPath string). Overrides all the format options but substitutions are still applied.                                                                                                                                                                                                         |
```

**Review by cg-tw — APPROVED** [review]

**Review by cg-tw — APPROVED** [review]

**Pull request merged**